servicetalk-opentelemetry-http/src/test/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpServerFilterTest.java

```diff
@@ -235,6 +236,11 @@ void testCaptureHeaders() throws Exception {
         }
     }
 
+    @Test
+    void verifyAsyncContextVisibility() throws Exception {
+        verifyServerFilterAsyncContextVisibility(new OpenTelemetryHttpServerFilter());
```

Comment:
@bryce-anderson I found the utility I was describing to you the other day and decided to submit a PR to demonstrate how to use it. We only have it for the server-side for now. It helps assert that filter in question does not break AsyncContext propagation.